code/game/objects/structures/crates_lockers/closets/secure/locker.dm

```diff
@@ -9,6 +9,21 @@
 	locked = TRUE
 	health = 200
 
+/obj/structure/closet/secure_closet/AltClick(mob/user)
+	. = ..()
+	var/obj/item/holding = user.get_active_hand()
+	if(!holding)
+		return
+	if(istype(holding, /obj/item/card/id))
```

Comment:
With the code below, this guard statement would be redundant; turn the istype into a guard statement

Reply:
best way to do that?

Reply:
if(!istype(...))
    return
var/obj/item/card/id/ID = holding

etc. etc.